packages/contracts/test/gitcoinTests/gitcoin.fixture.ts

```diff
@@ -30,7 +30,7 @@ export async function deployGitcoinMumbaiFixture() {
   const whale = <SignerWithAddress>await ethers.getImpersonatedSigner("0x9883d5e7dc023a441a01ef95af406c69926a0ab6");
 
   /* transfer 10 weth to signers */
-  for (let i = 0; i < 3; i++) {
+  for (let i = 0; i <4; i++) {
```

Comment:
Not sure why you're doing this. All Signers get a starting balance. Iirc this test suite does not fork from any chain?

Reply:
They get a starting balance of the native token but not of any ERC 20.

Reply:
Ah missed that, thanks